eas.json

```diff
@@ -16,7 +16,7 @@
       "extends": "base",
       "distribution": "internal",
       "android": {
-        "buildType": "app-bundle"
+        "buildType": "apk"
       },
```

Comment:
`app-bundle` should be used for `production`. `preview` cannot be `.aab` and should be `.apk`, since we cannot install `.aab` on the device. So it simply doesn't make sense